app/src/test/java/org/simple/clinic/drugs/EditMedicineUpdateTest.kt

```diff
@@ -31,4 +43,48 @@ class EditMedicineUpdateTest {
             hasEffects(RefillMedicines(patientUuid)),
         ))
   }
+
+  @Test
+  fun `when filled prescribed drugs are fetched and some of the drugs are updated on the same day, then show set edit medicine button state to save button`() {
+    val model = EditMedicinesModel.create(patientUuid, RefillMedicine)
+    val updateSpec = UpdateSpec(EditMedicinesUpdate(LocalDate.of(2020, 11, 18), ZoneOffset.UTC))
+    val prescribedDrugRecords = listOf(
+        TestData.prescription(uuid = UUID.fromString("4aec376e-1a8f-11eb-adc1-0242ac120002"), name = "Amlodipine1", createdAt = LocalDate.of(2020, 11, 18).toUtcInstant(userClock), updatedAt = LocalDate.of(2020, 11, 18).toUtcInstant(userClock)),
+        TestData.prescription(uuid = UUID.fromString("537a119e-1a8f-11eb-adc1-0242ac120002"), name = "Amlodipine2", createdAt = LocalDate.of(2020, 11, 18).toUtcInstant(userClock), updatedAt = LocalDate.of(2020, 11, 18).toUtcInstant(userClock)),
+        TestData.prescription(uuid = UUID.fromString("5ac2a678-1a8f-11eb-adc1-0242ac120002"), name = "Amlodipine3", createdAt = LocalDate.of(2020, 11, 18).toUtcInstant(userClock), updatedAt = LocalDate.of(2020, 11, 18).toUtcInstant(userClock)),
```

Comment:
We have alternate methods of constructing `LocalDate` and `Instant` which you should be using instead.

```kotlin
createdAt = Instant.parse("2020-11-18T00:00:00Z")
```

Reply:
Have you installed the [Simple IDE plugin](https://github.com/simpledotorg/simple-android-idea-plugin)? This adds a bunch of live templates which makes this a lot easier, since you can just type in `instant` and press `Enter` and this gets generated automatically.

Reply:
yes I did install it. Was not using it before tho.